Attorney_Online.pro

```diff
@@ -26,6 +26,28 @@ contains(DEFINES, DISCORD) {
   LIBS += -ldiscord-rpc
 }
 
+# Uncomment to use the static libs built in external
+#
+# git submodule update --init --recursive
+#
+# patch external/QtApng/.qmake.conf external/static-QtApng.patch
+# cd external/QtApng
+# qmake
+# make
+#
+# cd external/discord-rpc && mkdir build && cd build
+# cmake ..
+# make
+#
+# DEFINES += STATIC_LIB
+contains(DEFINES, STATIC_LIB) {
+  LIBS += -L$$PWD/external/QtApng/plugins/imageformats -L$$PWD/external/QtApng/lib
+  contains(DEFINES, DISCORD) {
+    LIBS += -L$$PWD/external/discord-rpc/build/src
+  }
+  LIBS += -l:libqapng.a -l:libqtapng_png.a -l:libqtapng_zlib.a
```

Comment:
Oh, I remember why I did not do static links of QtApng on Windows builds. There is a nasty issue where the entry point of the libpng self-test utility gets linked into the program and set as the main entry point of the application. Please check that this doesn't happen on Windows.

Reply:
I'm going to need help with this from someone. I don't have a windows dev environment setup for this.